next-tavla/src/Admin/components/DeleteModal/index.tsx

```diff
@@ -22,23 +25,28 @@ function DeleteModal({
             closeLabel="Avbryt sletting"
             className="flexColumn justifyStart alignCenter textCenter"
         >
-            <Heading1 className="text-rem-4">Slett tavle</Heading1>
+            <Image src={sheep} alt="" width={250} height={250} />
```

Comment:
Kunne vi vurdert noe annet enn å hardkode bredde og høyde her? Vi kunne kanskje gjort det lignende som vi har gjort i login modalen. Du kan se på Email siden her: `/app/(admin)/components/Login/Email.tsx`

Vi setter bredden og høyden til bildet til å være 50% av parent containeren med noen klasser:

```tsx
<Image src={musk} aria-hidden="true" alt="" className="h-50 w-50" />
```

Reply:
takk for innspill 🙌 satt height og width I classname